lrlex/src/lib/ctbuilder.rs

```diff
@@ -399,6 +402,7 @@ where
 
         outs.push_str(&format!(
             "{mod_vis} mod {mod_name} {{
+use cfgrammar::Span;
```

Comment:
I've just realised: we shouldn't *really* use global imports here. [Ideally we'd banish the other ones too, but not in this PR!] It's better if we always use `::cfgrammar::Span` in the generated code as then we can't have name clashes (i.e. the `::` syntax, though laborious, allow us to largely maintain hygiene when we copy the user's code in).

Reply:
This and the whitespace changes fixed in https://github.com/softdevteam/grmtools/pull/330/commits/d050bbfd082436697ce9852c71c3547fd71c50ef